src/components/shared/timeline-card.json

```diff
@@ -0,0 +1,28 @@
+{
+  "collectionName": "components_shared_timeline_cards",
+  "info": {
+    "displayName": "TimelineCard"
+  },
+  "options": {},
+  "attributes": {
```

Comment:
Question:
Shouldn't we have theme for card?

Reply:
The theme has been not mentioned in description. Let's discuss with Iryna, if we theme here.